pkg/cmd/pr/shared/survey.go

```diff
@@ -110,10 +110,20 @@ func BodySurvey(p Prompt, state *IssueMetadataState, templateContent string) err
 	return nil
 }
 
-func TitleSurvey(p Prompt, state *IssueMetadataState) error {
-	result, err := p.Input("Title", state.Title)
-	if err != nil {
-		return err
+func TitleSurvey(p Prompt, io *iostreams.IOStreams, state *IssueMetadataState) error {
+	var err error
+	result := ""
+	for result == "" {
+		result, err = p.Input("Title (required)", state.Title)
+		if err != nil {
+			return err
+		}
+		if result == "" {
+			colorizeRed := io.ColorScheme().ColorFromString("red")
+			msg := fmt.Sprintf("%s Title cannot be blank.", colorizeRed("X"))
+			// For some reason, only Fprintln and Println work here. I can't use Fprintf or Printf to eliminate the line above
```

Comment:
Haven't had a chance to investigate this, yet, but if anyone has insights to share I'd be quite thankful!

Reply:
I haven't completely compared this versus [other places `fmt.Fprintf(opts.IO.ErrOut`](https://github.com/search?q=repo%3Acli%2Fcli%20%22fmt.Fprintf(opts.IO.ErrOut%22&type=code) is being used but plenty of places are doing this 🤔

Reply:
The `io` argument is being passed in from `opts.io`, so as far as I understand it's technically already doing that.